### PR TITLE
[ServiceBus] March release notes

### PR DESCRIPTION
**Pull request by yunhaoling**

### CODE REVIEW

**Review by swathipil — APPROVED** [review]

File: releases/2021-03/python.md [hunk 1]
```diff
@@ -29,18 +29,31 @@ The Azure SDK team is pleased to make available the March 2021 client library re
 To install the latest beta version of the packages, copy and paste the following commands into a terminal:
 
 ```bash
-
+pip install azure-servicebus
```
Comment (xiangyan99) [inline releases/2021-03/python.md:32]: # ($> pip install ${PackageName}==${PackageVersion})

File: releases/2021-03/python.md [hunk 2]
```diff
@@ -29,18 +29,31 @@ The Azure SDK team is pleased to make available the March 2021 client library re
...
-[pattern]: # ($> pip install ${PackageName}==${PackageVersion})
-
 ## Feedback
 
 If you have a bug or feature request for one of the libraries, please post an issue to [GitHub](https://github.com/azure/azure-sdk-for-python/issues).
 
 ## Release highlights
 
-[pattern]: # (### ${PackageFriendlyName} ${PackageVersion} [Changelog]${ChangelogUrl}`n${HighlightsBody}`n)
+### Service Bus 7.1.0 [Changelog](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/servicebus/azure-servicebus/CHANGELOG.md#710-2021-03-09)
```
Comment (xiangyan99) [inline releases/2021-03/python.md:41]: Please pin to the release tag rather than using master

Comment (yunhaoling) [reply releases/2021-03/python.md]: thanks for calling it out! just updated

**Review by xiangyan99 — APPROVED** [review]

**Review by weshaggard — CHANGES_REQUESTED** [review]
Lets try to use https://github.com/Azure/azure-sdk/pull/2503 for the python changelogs.

**Comment by xiangyan99** [comment]
Updated in #2503

**Pull request closed without merging**